IBM/ibm-common-service-operator
# [CP3.0] reimplement operandrequest mutating webhook

How to test:

Setting up dev cs-operator image on your cluster:

1. ensure you have future catalogsource on your cluster
2. export REGISTRY=quay.io/yuchen_li1
3. run `make build-dev-image`
4. edit manager.yaml `image: quay.io/yuchen_li1/common-service-operator-amd64:dev`
5. run `make build-bundle-image`
6. run `make run-bundle`

Testing:
1. search for service `ibm-common-service-webhook`
2. search for mutating webhook `ibm-common-service-webhook-configuration`
3. create an operandrequest, mutating webhook will set the correct namespace for it.
